okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -811,7 +811,8 @@ public void shutdownNow(Status reason) {
       }
 
       for (OkHttpClientStream stream : pendingStreams) {
-        stream.transportState().transportReportStatus(reason, true, new Metadata());
+        stream.transportState().transportReportStatus(
+            reason, RpcProgress.MISCARRIED, true, new Metadata());
```

Comment:
This is in shutdownNow(). That's similar to this stream being explicitly cancelled. I don't think we should change this location, and I'm not aware of any need for it.

Reply:
If a connection fails to ack keep-alive, the transport will `shutdownNow`. I think the pending streams should have a chance to be routed to another connection.

Reply:
I see. Sounds good.